modules/custom/az_media/az_media_trellis/az_media_trellis.info.yml

```
@@ -5,3 +5,5 @@ core_version_requirement: ^10 || ^11 || ^12
 package: 'The University of Arizona - Experimental'
 lifecycle: experimental
 lifecycle_link: 'https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#experimental-features'
+dependencies:
+  - drupal:media_remote
```

I think it is like this 
```suggestion
  - media_remote:media_remote
```